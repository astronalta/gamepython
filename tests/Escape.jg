# jgtest exec on

main() Int {
    Boot::print_str("\a\b\t\n\v\f\r hello, world!\n")
    Boot::print_str("\x48\x45\x4c\x4c\x4f\x0a hello!\n")
    Boot::print_str("\o110\o105\o114\o114\o117\o012 hello!\n")
    Boot::print_str("\"")
    Boot::print_str("\#")
    Boot::print_str('\'')
    Boot::print_str('`')
    Boot::print_str('\n')
    ret 0
}
