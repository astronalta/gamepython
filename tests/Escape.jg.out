	
 hello, world!
HELLO
 hello!
HELLO
 hello!
"#'`
